BandolierForNPC.esp added

coco9515 opened the pull request:
I added my mod : [BandolierForNPC.esp](https://www.nexusmods.com/skyrimspecialedition/mods/19093) - and 4 others .esp to the list.

All of them required Dr_Bandolier.esp.
BandolierForNPC.esp is the main .esp but :
 - 3 of them are standalone compatibility patch, all the required .esp has been placed.
 - The last one is an optional add-on to my mod or Dr_Bandolier.esp.

coco9515 committed:
BandolierForNPC.esp added

Added BandolierForNPC.esp and 4 others .esp to the list.

Ortham commented:
If you've got a plugin in `req`, you don't need it in `after` too. If you remove the redundant entries, everything else looks fine.

coco9515 committed:
BandolierForNPC Updated

BandolierForNPC updated. Redundancy removed.

coco9515 commented:
Okay, I updated. Is everything okay now ?

MacSplody commented on masterlist.yaml:
```diff
@@ -5460,3 +5484,33 @@ plugins:
+  - name: 'BandolierForNPC - Cloaks of Skyrim Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - 'Cloaks.esp'
```
File masters automatically have a required entry, there is no need to add these as required if they're in the `esp` header.

MacSplody commented on masterlist.yaml:
```diff
@@ -5460,3 +5484,33 @@ plugins:
+  - name: 'BandolierForNPC - Winter is Coming Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
```
Same as above.

MacSplody commented on masterlist.yaml:
```diff
@@ -5460,3 +5484,33 @@ plugins:
+    req:
+      - 'Dr_Bandolier.esp'
+      - '1nivWICCloaks.esp'
+      - 'Cloaks.esp'
```
Same as above.

MacSplody commented on masterlist.yaml:
```diff
@@ -3857,6 +3857,30 @@ plugins:
         util: 'SSEEdit v3.2.2'
+  - name: 'BandolierForNPC.esp'
+    req:
+      - 'Dr_Bandolier.esp'
```
Also not needed.

MacSplody commented on masterlist.yaml:
```diff
@@ -3857,6 +3857,30 @@ plugins:
+      - Invent
+  - name: 'BandolierForNPC - Realistic Enchantements.esp'
+    req:
+      - 'Dr_Bandolier.esp'
```
Also not needed.

MacSplody reviewed (changes requested):
>If manually installing - choose only one

if only one of the below plugins can be installed then an `inc:` key should be used to ensure this.
`BandolierForNPC.esp`
`BandolierForNPC - Cloaks of Skyrim Patch`
`BandolierForNPC - Winter is Coming Patch`
`BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch`

so for example:

```yaml

  - name: 'BandolierForNPC.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093/'
        name: 'Bandolier - Bags and Pouches for NPC on Nexus Mods'
    inc:
      - 'BandolierForNPC - Cloaks of Skyrim Patch.esp'
      - 'BandolierForNPC - Winter is Coming Patch.esp'
      - 'BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch.esp'
    tag:
      - Names
      - Stats
```

MacSplody commented:
Also I usually mark mods that have masters in the wrong order as requiring sorting.
But since this is your mod, I'll leave it up to you if you want to fix that.

`BandolierForNPC - Realistic Enchantements.esp`

![image](https://user-images.githubusercontent.com/1944639/44171750-08253580-a0d3-11e8-83c1-b59901e898e0.png)

MacSplody commented on masterlist.yaml:
```diff
@@ -3857,6 +3857,33 @@ plugins:
+    req:
+      - 'Dr_Bandolier.esp'
+    after:
+      - 'Dr_Bandolier.esp'
```
- 'Dr_Bandolier.esp' is not needed in `after` as it's a master.

MacSplody commented on masterlist.yaml:
```diff
@@ -5460,3 +5487,42 @@ plugins:
+      - '1nivWICSkyCloaksPatch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC - Winter is Coming and Cloaks of Skyrim Patch
```
It's purely aesthetic but I usually keep `url` keys just under the `name` key for consistency.

```yaml

  - name: 'BandolierForNPC - Winter is Coming and Cloaks of Skyrim Patch.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093/'
        name: 'Bandolier - Bags and Pouches for NPC on Nexus Mods'

```

MacSplody reviewed (changes requested):
Okay so I need you to add `inc` entries for all standalone version so that only one version is used.

You can remove any Req & After entries where the plug-in is a master, as LOOT will handle requires and load after automatically for masters.

MacSplody commented on masterlist.yaml:
```diff
@@ -3857,6 +3857,30 @@ plugins:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19093'
+        name: Bandolier - Bags and Pouches for NPC
+    tag:
+      - Invent
```
You don't need the `Invent` tag as that is for changes to `NPC_` or `CONT` Container records.

The following Bash tags can be added as well to all your plug-ins.
```yaml
    tag:
      - Names
      - Stats
```

coco9515 committed:
BandolierForNPC New Update

Updated :
- all tag went from invent to names & stats
- inc added to all standalone esp
-rearange the esp listed in the after tag for realistic enchantements so it's the same as the recommended loot order
- updated the masters

coco9515 commented:
Ok, I think I did everything needed. Could you check ? Thank you for your time :)

MacSplody reviewed (approved):
Yeah looks good, Thanks for the contribution.
